Source/Morris.Reducible/Reducer.ConditionBuilder.cs

```diff
@@ -20,15 +20,11 @@ public class ConditionBuilder<TState, TDelta>
 		}
 
 		public ObjectResultMapper<TState, TSubState, TDelta> WhenReducedBy<TSubState>(
-			Func<TState, TSubState> subStateSelector,
-			Func<TSubState, TDelta, Result<TSubState>> reducer)
-		=>
-			new ObjectResultMapper<TState, TSubState, TDelta>(subStateSelector, reducer);
+			Func<TState, TSubState> subStateSelector, Func<TSubState, TDelta, Result<TSubState>> reducer) =>
```

Comment:
I prefer all of the original code in this case.

Reply:
Reverted